job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java

```diff
@@ -250,10 +250,14 @@ private synchronized void updateStatus() {
           mPlanInfo.setStatus(Status.FAILED);
           if (mPlanInfo.getErrorMessage().isEmpty()) {
             mPlanInfo.setErrorMessage("Task execution failed: " + info.getErrorMessage());
+            LOG.info("Job failed Id={} Error={}", mPlanInfo.getId(), info.getErrorMessage());
           }
           return;
         case CANCELED:
           if (mPlanInfo.getStatus() != Status.FAILED) {
+            if (mPlanInfo.getStatus() != Status.CANCELED) {
+              LOG.info("Job cancelled Id={}", mPlanInfo.getId());
```

Comment:
Maybe this should be `warn`? Also, is there something in `TaskInfo info` that would be useful to print to the log?

Reply:
I also think info here is fine. Also we have a Job Start info log that lets you know about some of the details.